releasenotes/notes/add-serializable-parametric-pulse-31490c4d2cc49ec6.yaml

```diff
@@ -0,0 +1,30 @@
+---
+features:
+  - |
+    New pulse base class :class:`~SymbolicPulse` has been added. This is a
+    drop-in replacement of the conventional :class:`~ParametricPulse` which has been deprecated.
+    In new base class, pulse envelope and parameter validation functions are
+    represented by sympy symbolic expression objects, which are lambidfied immediately when
+    the first instance of symbolic pulse subclass is created.
+    These new pulse subclasses can be QPY serialized through serialization mechanism of
```

Comment:
Should this note be added in the serialization PR instead?

Reply:
Fair enough. I removed the comment.